orc/src/main/java/org/apache/iceberg/orc/OrcMetrics.java

```diff
@@ -146,7 +146,8 @@ private static Metrics buildOrcMetrics(final long numOfRows, final TypeDescripti
       if (icebergColOpt.isPresent()) {
         final Types.NestedField icebergCol = icebergColOpt.get();
         final int fieldId = icebergCol.fieldId();
-        final MetricsMode metricsMode = effectiveMetricsConfig.columnMode(icebergCol.name());
+        final String colName = schema.findColumnName(icebergCol.fieldId());
```

Comment:
icebergCol.name() is  the unqualified column name

Reply:
Oh, good catch, @szehon-ho.

I think @yyanyy added a utility method for extracting a metrics mode by id and we use that in Parquet. I think we can use the same for consistency.

```
MetricsUtil.metricsMode(schema, effectiveMetricsConfig, fieldId)
```

Reply:
Didn't see it, good suggestion, done